src/components/typeAheadCheckboxes/typeaheadCheckboxes.tsx

```diff
@@ -196,12 +205,11 @@ const TypeaheadCheckboxes: React.FC<TypeaheadCheckboxesProps> = ({
           data-ouia-component-id="credentials_list_input"
         />
         <TextInputGroupUtilities>
-          {selected.length > 0 && (
+          {selectedOptions.length > 0 && (
             <Button
               variant="plain"
               onClick={() => {
                 setInputValue('');
-                setSelected([]);
```

Comment:
Blocking: "clear credentials" button is now broken, it does not do anything. It used to de-select everything that was selected.

It seems like something we could cover in unit test.

Reply:
ah, sorry. THis is fixed on the other PR by adding onChange([]) to properly clear all selections (rather than trying to use setSelected). The test I added covers this behavior, I am rebasing with main, the behavior should be restored.